packages/ui-library/src/components/select-field/stories.js

```diff
@@ -91,3 +99,19 @@ WithLabelSuffix.args = {
 	],
 };
 
+OptionsProp.parameters = {
+	docs: { description: { story: "Add options as an array of objects with `options` prop. Each object must contain `value` and `label` parameters. The displayed selected label will be updated automatically on change." } },
+};
+
+export const ChildrenProp = Template.bind( {} );
```

Comment:
Maybe this could be called `SelectField.Option` instead of `ChildrenProp`?
It hands you an "export" for ease of use. In case you do want to pass a `children` prop to the underlying Select.

Reply:
Done